.github/workflows/test.yml

```diff
@@ -4,26 +4,39 @@ on:
     branches:
       - master
 
+env:
+  # To run CI against unrelased go-ipfs or js-ipfs-* code (eg. wip PR),
+  # uncomment below and define git revisions in ./scripts/custom-runtime.sh
+  #IPFS_GO_EXEC: /tmp/go-ipfs/cmd/ipfs/ipfs
+  #IPFS_JS_EXEC: /tmp/js-ipfs/packages/ipfs/src/cli.js
+  #IPFS_JS_MODULE: /tmp/js-ipfs/packages/ipfs/dist/cjs/src/index.js
+  #IPFS_JS_HTTP_MODULE: /tmp/js-ipfs/packages/ipfs-http-client/dist/cjs/src/index.js
```

Comment:
This works and is documented ✔️

What's the advantage of doing this vs always testing against custom-runtime.sh, but sometimes checking out the release tag/branch and other times checking out a Git hash? We could pass two versions into the script (e.g. `custom-runtime.sh v0.10.0 v0.60.0`) and then have the script export environment variables.

Perhaps this is a pain and could be circumventing by passing the variables into the interop test. If so this seems like a fine change to make at a later date.

Reply:
The problem with `custom-runtime.sh` is that (as it is right now) builds dependencies from scratch, which takes  ~6min.
This would slow down CI in go-ipfs and js-ipfs repos where ipfs/interop also runs. 

We can make it smarter, but for now i just wanted to ship mvp so the next person has head-start.

Reply:
SGTM, thanks. I forgot how long the js-ipfs builds can take.